packages/components/src/components/hds/alert/title.hbs

```diff
@@ -2,4 +2,8 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<div class="hds-alert__title hds-font-weight-semibold" ...attributes>{{yield}}</div>
\ No newline at end of file
+{{! IMPORTANT: we removed any extra newlines before/after the `let` to reduce the issues with unexpected whitespaces (see https://github.com/hashicorp/design-system/pull/1652) }}
```

Comment:
This is fine (and I see this is already merged) but just to let you know, another way to remove whitespace within Ember HBS files is to use tilde symbols "~" (sometimes referred to as "squishies" in Ember)

An example of using squishies in our code base: https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/rich-tooltip/index.hbs

Reply:
Oh! Cool! Danger of copy/pasting from older files haha, I have another branch to update the default tags for Modal/Flyout so maybe I can clean that up in that PR.